NETWORK/N_0xC6FCEE21C6FCEE21.md

```diff
@@ -0,0 +1,22 @@
+---
+ns: NETWORK 
+---
+
+## _0xC6FCEE21C6FCEE21
+
+```c
+// 0xC6FCEE21C6FCEE21 
+void _0xC6FCEE21C6FCEE21(Any p0);
```

Comment:
Seems to be named `NETWORK_SET_TASK_CUTSCENE_INSCOPE_MULTIPLER` with float param for multiplier value. I bet it alters the scoping distance of `CTaskCutscene` for remote players. R* seems to reset to 1.0 when no longer needed.